packages/ui/src/components/Visualization/Canvas/CanvasFormTabs.tsx

```diff
@@ -0,0 +1,128 @@
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import { FunctionComponent, useCallback, useContext, useEffect, useMemo, useRef, useState } from 'react';
+import { EntitiesContext } from '../../../providers/entities.provider';
+import { SchemaBridgeProvider } from '../../../providers/schema-bridge.provider';
+import { getUserUpdatedPropertiesSchema, isDefined, setValue } from '../../../utils';
+import { CustomAutoForm, CustomAutoFormRef } from '../../Form/CustomAutoForm';
+import { DataFormatEditor } from '../../Form/dataFormat/DataFormatEditor';
+import { LoadBalancerEditor } from '../../Form/loadBalancer/LoadBalancerEditor';
+import { StepExpressionEditor } from '../../Form/stepExpression/StepExpressionEditor';
+import { UnknownNode } from '../Custom/UnknownNode';
+import './CanvasFormTabs.scss';
+import { CanvasNode } from './canvas.models';
+
+interface CanvasFormTabsProps {
+  selectedNode: CanvasNode;
+}
+
+export enum FormTabsModes {
+  ALL_FIELDS = 'All Fields',
+  USER_MODIFIED = 'User Modified',
+}
```

Comment:
It's better to export this enum in a separate file, it can be in this `Canvas` folder but separately from the `CanvasFormTabs` component so the live-reload it's not affected.

https://github.com/vitejs/vite-plugin-react-swc?tab=readme-ov-file#consistent-components-exports